src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,20 @@
+package duke.task;
+
+public class Deadline extends Task {
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    /**
```

Comment:
I think the javadoc for `toString` here is not necessary since you already have the `@Override` here which explains that you are overriding the `toString()` from the parent class.